docker-compose.yml

```diff
@@ -7,23 +7,36 @@ services:
     volumes:
       - imaginethis-db-vol:/var/lib/postgresql/data
     ports:
-      - 5432:5432
+      - 5432
 
   imaginethis-backend:
     build: ./ImagineThisServer
     container_name: imaginethis-backend
     environment:
-      - DB_SERVER=imaginethis-db
+      - IMAGINE_THIS_DB_HOST=postgresql://imaginethis-db
+      - IMAGINE_THIS_DB_PORT=5432
+      - IMAGINE_THIS_DB_NAME=
+      - IMAGINE_THIS_USERNAME=
+      - IMAGINE_THIS_PASSWORD=
+      - CORS_ORIGIN=
```

Comment:
Can't we set here the 2 options we'll use most heavily: localhost and 213.168.248.64?

Reply:
I actually don't think that the environment variable should be hardcoded, but that's just my thought

Reply:
Ok, let's keep it that way then